examples/pipelines/starcoder/pipeline.py

```diff
@@ -1,5 +1,6 @@
 """Pipeline used to create the dataset to train the StarCoder model."""
 
+import argparse
```

Comment:
Why was this added? I think we should be able to remove the `if __name__ = "__main__":` block at the bottom of the file, no?